Bukkit-1.14/pom.xml

```diff
@@ -108,8 +112,8 @@
             <version>0.9.0-SNAPSHOT</version>
         </dependency>
         <dependency>
-            <groupId>org.spigotmc</groupId>
-            <artifactId>spigot</artifactId>
+            <groupId>com.destroystokyo.paper</groupId>
+            <artifactId>paper</artifactId>
```

Comment:
Time to add a CI job for Paper wee

Reply:
We don't want to use Paper. We're a Spigot supporting community. We should only rely on Paper-API.